2020-03-12 10:56:04,988 : kivy : Factory: 184 symbols loaded
2020-03-12 10:56:05,088 : kivy : Image: Providers: img_tex, img_dds, img_sdl2, img_pil, img_gif (img_ffpyplayer ignored)
2020-03-12 10:56:05,147 : kivy : VideoGstplayer: Using Gstreamer 1.16.0.0
2020-03-12 10:56:05,147 : kivy : Video: Provider: gstplayer
2020-03-12 10:56:05,216 : kivy : Loading MPF-MC controller
2020-03-12 10:56:05,216 : mpfmc : Mission Pinball Framework Media Controller v0.53.0
2020-03-12 10:56:05,216 : mpfmc : Mission Pinball Framework Game Engine v0.53.2
2020-03-12 10:56:05,217 : mpfmc : Machine path: E:\mpf\wows\wows
2020-03-12 10:56:05,218 : ConfigProcessor : Config file in cache changed: E:\mpf\wows\wows\config\config
2020-03-12 10:56:05,218 : ConfigProcessor : Loading config from file c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\mcconfig.yaml.
2020-03-12 10:56:05,256 : ConfigProcessor : Loading config: c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\mcconfig.yaml
2020-03-12 10:56:05,263 : ConfigProcessor : Loading config from file E:\mpf\wows\wows\config\config.
2020-03-12 10:56:05,377 : ConfigProcessor : Loading config: E:\mpf\wows\wows\config\config
2020-03-12 10:56:05,380 : ConfigProcessor : Config file cache created: C:\Users\schol\AppData\Local\Temp\398a03908486e3c8525902f452489fb5.mpf_cache
2020-03-12 10:56:05,380 : mpfmc : Starting clock at 60.0Hz
2020-03-12 10:56:05,391 : kivy : Text: Provider: sdl2
2020-03-12 10:56:05,428 : kivy : Window: Provider: sdl2
2020-03-12 10:56:05,873 : kivy : GL: Using the "OpenGL" graphics system
2020-03-12 10:56:05,879 : kivy : GL: GLEW initialization succeeded
2020-03-12 10:56:05,879 : kivy : GL: Backend used <glew>
2020-03-12 10:56:05,879 : kivy : GL: OpenGL version <b'4.6.0 NVIDIA 432.00'>
2020-03-12 10:56:05,879 : kivy : GL: OpenGL vendor <b'NVIDIA Corporation'>
2020-03-12 10:56:05,879 : kivy : GL: OpenGL renderer <b'GeForce GTX 1060 6GB/PCIe/SSE2'>
2020-03-12 10:56:05,879 : kivy : GL: OpenGL parsed version: 4, 6
2020-03-12 10:56:05,879 : kivy : GL: Shading version <b'4.60 NVIDIA'>
2020-03-12 10:56:05,879 : kivy : GL: Texture max size <32768>
2020-03-12 10:56:05,879 : kivy : GL: Texture max units <32>
2020-03-12 10:56:05,897 : kivy : Window: auto add sdl2 input provider
2020-03-12 10:56:05,897 : kivy : Window: virtual keyboard not allowed, single mode, not docked
2020-03-12 10:56:05,904 : SoundSystem : SoundSystem: Using default 'sound_system' settings
2020-03-12 10:56:06,095 : AudioInterface : Initialized
2020-03-12 10:56:06,096 : SoundSystem : No audio tracks are specified in your machine config file. a track named 'default' has been created.
2020-03-12 10:56:06,097 : kivy : Base: Start application main loop
2020-03-12 10:56:06,097 : EventManager : Event: ======'display_window_initialized'====== Args={}
2020-03-12 10:56:06,097 : EventManager : Event: ======'display_dmd_initialized'====== Args={}
2020-03-12 10:56:06,148 : mpfmc : Display: Setting 'dmd' as default display
2020-03-12 10:56:06,154 : EventManager : Event: ======'displays_initialized'====== Args={}
2020-03-12 10:56:06,155 : EventManager : Event: ======'slide_window_blank_created'====== Args={}
2020-03-12 10:56:06,155 : EventManager : Event: ======'display_window_ready'====== Args={}
2020-03-12 10:56:06,155 : EventManager : Event: ======'slide_dmd_blank_created'====== Args={}
2020-03-12 10:56:06,155 : EventManager : Event: ======'display_dmd_ready'====== Args={}
2020-03-12 10:56:06,163 : EventManager : Event: ======'init_phase_1'====== Args={}
2020-03-12 10:56:06,166 : EventManager : Event: ======'mc_ready'====== Args={}
2020-03-12 10:56:06,166 : EventManager : Event: ======'init_phase_2'====== Args={}
2020-03-12 10:56:06,168 : ConfigProcessor : Cache file not found: C:\Users\schol\AppData\Local\Temp\fa61b88f86d6e414c9b51ac1d3e19568.mpf_cache
2020-03-12 10:56:06,168 : ConfigProcessor : Loading config from file E:\mpf\wows\wows\modes\hc_multiball_super\config\hc_multiball_super.yaml.
2020-03-12 10:56:06,175 : kivy : Base: Leaving application in progress...
2020-03-12 10:56:06,175 : kivy : Exception while processing RegisteredHandler(callback=<bound method ModeController._load_modes of <mpfmc.core.mode_controller.ModeController object at 0x000001D7321EAFD0>>, priority=1, kwargs={}, key=UUID('1f160572-c652-4eda-bd14-4227e435dd81'), condition=None, blocking_facility=None) for event init_phase_2
Traceback (most recent call last):
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 193, in load
    config = self.process(f)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 218, in process
    return yaml.load(data_string, Loader=MpfLoader)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\ruamel\yaml\main.py", line 935, in load
    return loader._constructor.get_single_data()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\ruamel\yaml\constructor.py", line 111, in get_single_data
    return self.construct_document(node)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\ruamel\yaml\constructor.py", line 121, in construct_document
    for _dummy in generator:
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\ruamel\yaml\constructor.py", line 713, in construct_yaml_map
    value = self.construct_mapping(node)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 135, in construct_mapping
    format(key, key_node.start_mark))
KeyError: 'Key "event_player" was defined multiple times in config   in "E:\\mpf\\wows\\wows\\modes\\hc_multiball_super\\config\\hc_multiball_super.yaml", line 41, column 1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 665, in _run_handlers
    result = handler.callback(**merged_kwargs)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mode_controller.py", line 59, in _load_modes
    self.mc.modes[mode] = self._load_mode(mode)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mode_controller.py", line 132, in _load_mode
    store_to_cache=self.mc.options['create_config_cache'], ignore_unknown_sections=True)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 113, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 169, in _load_config_file_and_return_loaded_files
    config = FileManager.load(filename, expected_version_str, True)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\file_manager.py", line 98, in load
    return interface.load(file, verify_version, halt_on_error)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 207, in load
    raise ValueError(msg)
ValueError: Error found in file E:\mpf\wows\wows\modes\hc_multiball_super\config\hc_multiball_super.yaml: 'Key "event_player" was defined multiple times in config   in "E:\\mpf\\wows\\wows\\modes\\hc_multiball_super\\config\\hc_multiball_super.yaml", line 41, column 1'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\commands\mc.py", line 213, in __init__
    thread_stopper=thread_stopper).run()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\app.py", line 855, in run
    runTouchApp()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\base.py", line 504, in runTouchApp
    EventLoop.window.mainloop()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\core\window\window_sdl2.py", line 747, in mainloop
    self._mainloop()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\core\window\window_sdl2.py", line 479, in _mainloop
    EventLoop.idle()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\base.py", line 339, in idle
    Clock.tick()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\kivy\clock.py", line 591, in tick
    self._process_events()
  File "kivy\_clock.pyx", line 384, in kivy._clock.CyClockBase._process_events
  File "kivy\_clock.pyx", line 414, in kivy._clock.CyClockBase._process_events
  File "kivy\_clock.pyx", line 412, in kivy._clock.CyClockBase._process_events
  File "kivy\_clock.pyx", line 167, in kivy._clock.ClockEvent.tick
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\uix\display.py", line 205, in _displays_initialized
    self.mc.displays_initialized()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mc.py", line 336, in displays_initialized
    self._init()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpfmc\core\mc.py", line 380, in _init
    self.events.process_event_queue()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 752, in process_event_queue
    **event[3])
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 711, in _process_event
    result = self._run_handlers(event, ev_type, kwargs)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 667, in _run_handlers
    raise Exception("Exception while processing {} for event {}".format(handler, event)) from e
Exception: Exception while processing RegisteredHandler(callback=<bound method ModeController._load_modes of <mpfmc.core.mode_controller.ModeController object at 0x000001D7321EAFD0>>, priority=1, kwargs={}, key=UUID('1f160572-c652-4eda-bd14-4227e435dd81'), condition=None, blocking_facility=None) for event init_phase_2
2020-03-12 10:56:06,276 : kivy : Stopping child threads... (1 remaining)
2020-03-12 10:56:07,182 : kivy : All child threads stopped.
